ILRepack/RepackImporter.cs

```diff
@@ -119,7 +119,11 @@ public MethodReference Import(MethodReference reference, IGenericParameterProvid
 
         public TypeDefinition Import(TypeDefinition type, Collection<TypeDefinition> col, bool internalize)
         {
-            _logger.Verbose("- Importing " + type);
+			if (CanInclude(type) == false) {
```

Comment:
Current code uses spaces (4 spaces) instead of tab, let's keep consistency

Reply:
I prefer tabs in all my projects.
Can you upload an editor config file to ensure that spacing consistency (as well as others) so I can follow it?